elastic/kibana
# [Global Search] Return focus to live region after page re-render even when left nav is open

## Summary

Closes https://github.com/elastic/kibana/issues/186423 

This code is interfering with the focus to the ScreenReaderLive region which is needed to announce page renders for assistive technologies. 

### After

https://github.com/user-attachments/assets/8b6141d1-201c-47fd-9c03-992ae7f275ce

Note the active element after the click is now the EuiScreenReaderOnly element to announce the page title

### To Test

Add `document.activeElement` in the developer tools console to see what element is active when testing keyboard navigation. After clicking a link, it should return focus to the ScreenReaderLive component to announce the page title to the user.
